dev-image.aib.yml

```
@@ -0,0 +1,114 @@
+    - dnf
+    - tar
+    - osbuild
+    - python3.12
+    - python3.12-pip
+    - podman
+    - automotive-image-builder
+
+  add_files:
+    - path: /oc-arm64-rhel9.tar.gz
+      url: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.18/openshift-client-linux-arm64-rhel9.tar.gz
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)